source/web_api/serverless.yml

```diff
@@ -4,21 +4,21 @@ provider:
   runtime: python2.7
   stage: ${opt:stage}
   region: us-east-1
-  role: #ROLE
+  role: ${ssm:/${opt:ssm-store}/${opt:stage}/role}
   stackTags:
-    # TEAM : TEAM EMAIL
+    TEAM: ${ssm:/${opt:ssm-store}/team}
   environment:
-    DATABASE_NAME: ${file(./env/env.${self:provider.stage}.yml):DATABASE_NAME}
-    REDSHIFT_PORT: ${file(./env/env.${self:provider.stage}.yml):REDSHIFT_PORT}
-    AWS_RS_USER: ${file(./env/env.${self:provider.stage}.yml):AWS_RS_USER}
-    AWS_RS_PASS: ${file(./env/env.${self:provider.stage}.yml):AWS_RS_PASS}
-    CLUSTER_ENDPOINT: ${file(./env/env.${self:provider.stage}.yml):CLUSTER_ENDPOINT}
-    JH_JIRAURL: ${file(./env/env.${self:provider.stage}.yml):JH_JIRAURL}
-    JH_USER: ${file(./env/env.${self:provider.stage}.yml):JH_USER}
-    JH_PASS: ${file(./env/env.${self:provider.stage}.yml):JH_PASS}
-    GIT_API_USER: ${file(./env/env.${self:provider.stage}.yml):GIT_API_USER}
-    GIT_API_KEY: ${file(./env/env.${self:provider.stage}.yml):GIT_API_KEY}
-    ENV: ${file(./env/env.${self:provider.stage}.yml):ENV}
+    DATABASE_NAME: ${ssm:/${opt:ssm-store}/redshift/${opt:stage}/database_name}
+    REDSHIFT_PORT: ${ssm:/${opt:ssm-store}/redshift/${opt:stage}/port}
+    AWS_RS_USER: ${ssm:/${opt:ssm-store}/redshift/${opt:stage}/username~true}
+    AWS_RS_PASS: ${ssm:/${opt:ssm-store}/redshift/${opt:stage}/password~true}
+    CLUSTER_ENDPOINT: ${ssm:/${opt:ssm-store}/redshift/${opt:stage}/cluster_endpoint}
+    JH_JIRAURL: ${ssm:/${opt:ssm-store}/jira/${opt:stage}/host_url}
+    JH_USER: ${ssm:/${opt:ssm-store}/jira/${opt:stage}/username~true}
+    JH_PASS: ${ssm:/${opt:ssm-store}/jira/${opt:stage}/password~true}
+    GIT_API_USER: ${ssm:/${opt:ssm-store}/git/${opt:stage}/api_user~true}
+    GIT_API_KEY: ${ssm:/${opt:ssm-store}/git/${opt:stage}/api_key~true}
+    ENV: ${opt:ssm-store}
```

Comment:
This is incorrectly set as `${opt:ssm-store}` and should instead be `${opt:stage}`